tests/test_marker.py

```diff
@@ -41,32 +43,32 @@ def test_pixel_corners(self):
         )
         self.assertEqual(bl, (border_size, self.MARKER_SIZE + border_size - 1))
 
-    def test_pixel_centre(self):
+    def test_pixel_centre(self) -> None:
         tl, _, br, _ = self.marker.pixel_corners
         self.assertEqual(self.marker.pixel_centre, (139, 139))
 
-    def test_distance(self):
+    def test_distance(self) -> None:
         self.assertEqual(self.marker.distance, 992)
 
-    def test_orientation(self):
+    def test_orientation(self) -> None:
         rot_x, rot_y, rot_z = self.marker.orientation
         self.assertEqual(int(rot_x), -3)
         self.assertEqual(int(rot_y), 0)
         self.assertEqual(int(rot_z), 0)
 
-    def test_cartesian_coordinates(self):
+    def test_cartesian_coordinates(self) -> None:
         x, y, z = self.marker.cartesian
         self.assertEqual(int(x), 49)
         self.assertEqual(int(y), 24)
         self.assertEqual(int(z), 991)
 
-    def test_spherical_coordinates(self):
-        rot_x, rot_y, dist = self.marker.spherical
-        self.assertEqual(dist, self.marker.distance)
-        self.assertEqual(rot_x, approx(0.025, abs=1e-3))
-        self.assertEqual(rot_y, approx(0.05, abs=1e-3))
+    def test_spherical_coordinates(self) -> None:
+        rot_x, rot_y, dist = self.marker.spherical  # type: ignore
```

Comment:
You shouldn't have to ignore these types, especially as `spherical` has a known type

Reply:
`cached_property` breaks it unfortunately.

Reply:
Also, worth noting that `mypy` will tell you if you are using a `type: ignore` when you don't need to, so tests would fail if this weren't the case.

Reply:
I'd recommend using the `Spherical` type directly here, so it's more explicit.

Alternatively, working out why `cached_property` breaks it (it definitely shouldnt!) would be even better!

Reply:
e868eb5